### ISSUE TITLE
Change position of HUD...

### ISSUE THREAD

**Issue by FirenzeIT** [yumi-modding/FS22_CombineXP#48]
Hello @yumi-modding , how can i change the position of HUD?? Because, when i have 4 lines of forage harvester or any else, the HUD covered the rest of lines... i add below a screenshot... Thank you.

![Screenshot 2024-02-09 060415](https://github.com/yumi-modding/FS22_CombineXP/assets/113168067/f5adfca2-177a-4109-a128-fc55c9933791)

I need to move this hud on up, but not up to speedmeter, need to move in same position but more up than this...

![fsScreen_2024_02_09_07_16_28](https://github.com/yumi-modding/FS22_CombineXP/assets/113168067/9606a886-d7fd-431d-9a1e-81f6e4e42f59)

up on all the lines

**Comment by yumi-modding** [comment]
Hello

This topic should help if you have not seen it yet
https://github.com/yumi-modding/FS22_CombineXP/issues/46#issuecomment-1824061100

**Comment by FirenzeIT** [comment]
Yes i see but i need to move them only upper the lines bar, not in other
part.. Example up to the lines, always in same position (left to speedmeter) but a little up, because when i have 4 lines bars, the hud cover it...

Il Ven 9 Feb 2024, 21:10 yumi-modding ***@***.***> ha scritto:
